src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -394,6 +394,31 @@ public void execute_select_jumpsToCorrectTask() throws Exception {
     }
 
     @Test
+    public void execute_MarkInvalidArgsFormat_errorMessageShown() throws Exception {
```

Comment:
these are what I wanna do, I created the test cases for the mark command. They are pretty similar to the delete command as the command format is similar.
They will test for invalid Index and wrong index.